### PR TITLE
add GCP project ID in manage domains

### PR DESCRIPTION
**Pull request by Raja0623**
# Description

# Contribution Checklist:
- [ ] **The pull request does not introduce any breaking changes**
- [ ] **I have read the [contribution guidelines](https://github.com/AthenZ/athenz/blob/master/CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request.**

### CODE REVIEW

File: ui/src/pages/domain/manage.js [hunk 1]
```diff
@@ -115,6 +117,7 @@ export async function getServerSideProps(context) {
             manageDomains: domains[1],
             pending: domains[2],
             _csrf: domains[3],
+            testfive:domains[4],
```
Comment (MartinTrojans) [inline ui/src/pages/domain/manage.js:120]: What is testfive?

File: ui/src/components/domain/ManageDomains.js [hunk 2]
```diff
@@ -401,6 +406,15 @@ class ManageDomains extends React.Component {
                                   />
                               ) : null}
                           </TDStyled>
+
+                          <TDStyled color={color} align={center}>
+                              {item.domain.account ? item.domain.account : ''}
+                          </TDStyled>
+
```
Comment (MartinTrojans) [inline ui/src/components/domain/ManageDomains.js:413]: Can you remove the redundant lines?

File: ui/src/components/domain/ManageDomains.js [hunk 3]
```diff
@@ -491,6 +508,11 @@ class ManageDomains extends React.Component {
     }
 }
 
+
```
Comment (MartinTrojans) [inline ui/src/components/domain/ManageDomains.js:511]: Can you remove the redundant lines?

**Comment by jimmytsang** [comment]
Can we add a functional test to make sure gcp project id exists?

File: ui/src/components/domain/ManageDomains.js [hunk 4]
```diff
@@ -401,6 +406,15 @@ class ManageDomains extends React.Component {
                                   />
                               ) : null}
                           </TDStyled>
+
+                          <TDStyled color={color} align={center}>
+                              {item.domain.account ? item.domain.account : ''}
```
Comment (chandrasekhar1996) [inline ui/src/components/domain/ManageDomains.js:411]: what is this account after the delete icon?

File: ui/src/pages/domain/manage.js (hunk 1 above)
Comment (Raja0623) [reply ui/src/pages/domain/manage.js:120]: It's just another demo API for testing, I'm going to remove it.

File: ui/src/pages/domain/manage.js [hunk 5]
```diff
@@ -115,6 +117,7 @@ export async function getServerSideProps(context) {
             manageDomains: domains[1],
             pending: domains[2],
             _csrf: domains[3],
+            test:domains[4],
```
Comment (chandrasekhar1996) [inline ui/src/pages/domain/manage.js:120]: where are you using this test?

File: ui/src/components/domain/ManageDomains.js [hunk 6]
```diff
@@ -32,23 +32,19 @@ import {
     selectBusinessServicesAll,
     selectTimeZone,
 } from '../../redux/selectors/domains';
-
```
Comment (chandrasekhar1996) [inline ui/src/components/domain/ManageDomains.js:35]: don't remove existing new lines

Comment (chandrasekhar1996) [reply ui/src/components/domain/ManageDomains.js:35]: applies for the entire file

File: ui/src/components/domain/ManageDomains.js (hunk 4 above)
Comment (Raja0623) [reply ui/src/components/domain/ManageDomains.js:385]: it has to do with the alignment of the "account" column.

Comment (chandrasekhar1996) [reply ui/src/components/domain/ManageDomains.js:385]: we already display account in line 380, not the right way to align columns

File: ui/src/pages/domain/manage.js (hunk 5 above)
Comment (Raja0623) [reply ui/src/pages/domain/manage.js:120]: I am using it for testing purpose, I using "testfive:domains[4]" to test my API.

File: ui/src/components/domain/ManageDomains.js (hunk 4 above)
Comment (Raja0623) [reply ui/src/components/domain/ManageDomains.js:385]: I see that it was already mentioned in the earlier lines, I fixed the error.

File: ui/src/pages/domain/manage.js (hunk 5 above)
Comment (jimmytsang) [reply ui/src/pages/domain/manage.js:120]: sorry i'm also a bit confused about this change, can you help me understand as well raja?

**Comment by jimmytsang** [comment]
please address chandu's comments

File: ui/src/components/domain/ManageDomains.js (hunk 6 above)
Comment (Raja0623) [reply ui/src/components/domain/ManageDomains.js:35]: Got it.

**Review by chandrasekhar1996 — APPROVED** [review]

**Pull request merged**